functions/utils/getApolloClient.ts

```diff
@@ -0,0 +1,21 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+const GRAPHQL_ENDPOINT = 'https://api.uniswap.org/v1/graphql'
+
+export const getApolloClient = () => {
+  return new ApolloClient({
+    connectToDevTools: true,
+    uri: GRAPHQL_ENDPOINT,
+    headers: {
+      'Content-Type': 'application/json',
+      Origin: 'https://app.uniswap.org',
```

Comment:
Do you _need_ an origin?

Reply:
yeah otherwise it gets autofiltered by backend